nautobot/core/views/mixins.py

```diff
@@ -499,9 +520,22 @@ def get_queryset(self):
         queryset = super().get_queryset()
         return queryset.restrict(self.request.user, self.get_action())
 
+    def get_action_map(self):
+        """
+        Return a mapping of action names to their corresponding permissions.
+        This is used to determine the permissions required for each action.
+        """
+        return PERMISSIONS_ACTION_MAP
```

Comment:
I don't see anything that's overriding this method now, is it still even needed?